### DISCUSSION TITLE
3.1.4 TSML UI Meeting Feedback

### DISCUSSION THREAD

**Discussion by Grateful-Viv** [code4recovery/12-step-meeting-list#767] [Q&A]
Piggybacking on #762, the meeting feedback email is definitely a work in progress. It is going to cause confusion for people submitting meeting changes to see ​Public URL: https://domain-name/?meeting=name-of-meeting
Edit URL: row xxx

Is there any way to at least remove the Edit URL line?

Is there any way to offer admin option to opt out entirely? Is that still User Feedback Emails? 

Is there a way to have admin create a link to customized form?

**Comment by joshreisner** [comment]
Hi @Grateful-Viv 

I've [created an issue to get rid of row XX](https://github.com/code4recovery/tsml-ui/issues/223) and [reopened an issue](https://github.com/code4recovery/12-step-meeting-list/issues/438) to get an actual WordPress edit URL in the email message instead

Admins can opt out entirely the same way they would in TSML, which is to remove the User Feedback Emails

There is a way to link to a customized form! from the readme:

```
$tsml_feedback_url = "https://domain.com?meeting={{slug}}";
$tsml_feedback_url = "https://domain.com?meeting={{id}}";
$tsml_feedback_url = "mailto:office@domain.com?subject={{slug}}";
```

it's perhaps missing the instruction that you can define a `$tsml_feedback_url` variable in your theme's functions.php with a link, and use `{{id}}` and `{{slug}}` as "variables" in the url. but if you wanted to link them to a page called `/feedback` it would be as simple as adding the line:

```
$tsml_feedback_url = "/feedback";
```

**Reply by Grateful-Viv** [reply]
If I am understanding you correctly, I can, for example, create a form on a page called `feedback` and add this line somewhere in my theme's child functions.php `$tsml_feedback_url = "/feedback";` and that would be all that is needed?

**Reply by joshreisner** [reply]
yes, should be
